hack/test-e2e-local.sh

```diff
@@ -29,6 +29,9 @@ ginkgo_flags=
 if [ -n "${CI:-}" -a -n "${ARTIFACTS:-}" ]; then
   mkdir -p "$ARTIFACTS"
   ginkgo_flags="--output-dir=$ARTIFACTS --junit-report=junit.xml"
+  if [ "${JOB_TYPE:-}" != "periodic" ]; then
+    ginkgo_flags+=" --failFast"
```

Comment:
```suggestion
    ginkgo_flags+=" --fail-fast"
```
see [here](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/8619/pull-gardener-e2e-kind-migration-ha-single-zone/1711367722157215744#1:build-log.txt%3A777) 😅